src/ontology/cl-edit.owl

```diff
@@ -32727,6 +32731,31 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:7107724") oboInOwl:hasRe
+AnnotationAssertion(terms:contributor obo:CL_4052012 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052012 "2024-09-30T11:10:15Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31320652") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4052012 "inTC")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31320652") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4052012 "interna TC")
+AnnotationAssertion(rdfs:label obo:CL_4052012 "interna theca cell")
+EquivalentClasses(obo:CL_4052012 ObjectIntersectionOf(obo:CL_0000503 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000157)))
+SubClassOf(obo:CL_4052012 obo:CL_0000593)
+SubClassOf(obo:CL_4052012 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_4052010))
+DisjointClasses(obo:CL_4052012 obo:CL_4052013)
+
+# Class: obo:CL_4052013 (externa theca cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31320652") Annotation(oboInOwl:hasDbXref "PMID:32530882") Annotation(oboInOwl:hasDbXref "PMID:36758341") Annotation(oboInOwl:hasDbXref "Wikipedia:Theca_externa") obo:IAO_0000115 obo:CL_4052013 "A specialized theca cell that forms the outer layer of the theca surrounding the ovarian follicle, appearing at the antral follicle. Originating from progenitor theca cells, theca externa cell is characterized by its fibroblast-like appearance and primarily function to provide structural support to the developing follicle. This cell produces collagen fibers and extracellular matrix components such as Col1a1 and Col1a2.")
+AnnotationAssertion(terms:contributor obo:CL_4052013 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052013 "2024-09-30T11:10:15Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31320652") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4052013 "exTC")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31320652") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4052013 "externa TC")
+AnnotationAssertion(rdfs:label obo:CL_4052013 "externa theca cell")
+EquivalentClasses(obo:CL_4052013 ObjectIntersectionOf(obo:CL_0000503 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000156)))
+SubClassOf(obo:CL_4052013 obo:CL_0002320)
```

Comment:
This is not needed since it is inherited (theca cell is a stromal cell, which is a connective tissue cell)